### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by jdinius-dev** [OpenFAST/openfast#496]
**Bug description**
I am attempting to run the regression tests, but I am getting multiple failures, mostly of the tests involving the 5 MW turbine. I have checked the plots and the errors are substantial.

**To Reproduce**
<Update the following list with your specific information.>
1. compile OpenFAST normally as described in the readthedocs, and set up the testing suite
2. Run cmake .. -DBUILD_TESTING=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo
make install
ctest
3. Checking the plots reveals that the tests 

**Expected behavior**
I expect the results to be similar to the baselines, but the tests that failed are significantly off. An example is shown below. 

**Screenshots, if applicable**
![Screenshot_2020-07-15 SWRT_YFree_VS_WTurb Summary(1)](https://user-images.githubusercontent.com/66981721/87555460-b72d6b80-c672-11ea-9efe-60f15f7df751.jpg)

**OpenFAST Version**
<Please provide as much detail as possible including git commit. The best information is a screenshot of the OpenFAST system description that prints when running OpenFAST:>

```
**************************************************************************************************
 OpenFAST

 Copyright (C)  National Renewable Energy Laboratory
 Copyright (C)  Envision Energy USA LTD

 This program is licensed under Apache License Version 2.0 and comes with ABSOLUTELY NO WARRANTY.
 See the "LICENSE" file distributed with this software for details.
 **************************************************************************************************

 OpenFAST-v2.3.0
 Compile Info:
  - Architecture: 64 bit
  - Precision: double
  - Date: Jul 14 2020
  - Time: 08:20:13
 Execution Info:
  - Date: 07/15/2020
  - Time: 08:16:07-0600
``

**System Information**
 - OS: Ubuntu 20.04
 - Compiler: GNU Fortran 9.3.0 
 - Compiler settings: DBUILD_TESTING=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo

**Additional context**
I have attempted solutions from other bug reports but have had no success. I am new to OpenFAST and only moderately experienced with Linux/Ubuntu commands. Any help is greatly appreciated

**Comment by bjonkman** [comment]
Can you show the results from the 5MW turbines? The SWRT model you are showing has some known instabilities that shift with almost every recompile (your results here aren't unexpected), but the 5MW results should be stable.

**Comment by jdinius-dev** [comment]
Sorry for the slow response, I ran into some major computer trouble and had to rerun my tests (still running now actually). Here's one of the 5MW tests that failed. 
![Screenshot_2020-07-15 5MW_OC3Mnpl_DLL_WTurb_WavesIrr Summary](https://user-images.githubusercontent.com/66981721/87587340-2e78f480-c69f-11ea-9d52-462410d07b37.png)

**Comment by andrew-platt** [comment]
I noticed that your `Wave1Elev` is different than the baseline.  This is probably because the random number generator used in gcc/gfortran 9.3 is different than the random number generator used in gcc/gfortran 7.4 that was used to generate the baseline.

We are exploring options for replacing `ranlux` so that different versions of gcc/gfortran will use the same random number generator. See bug report #89

**Comment by jdinius-dev** [comment]
OK, thanks for the help. This discrepency seems to be present in my other 5 MW tests as well. Should I change the gfortran version to 7.4 (and if so how would I do that) or do I wait for a bug-fix?

**Comment by andrew-platt** [comment]
I would expect this to appear in any of the tests that include hydrodynamic forces (`Waves` is present in the name, but not `WavesReg`).  If the only tests failing are these ones and the SWRT, then I think everything is working as expected on your system.

I don't know the expected timeline for the random number generator update, so you likely don't want to wait for that.

**Comment by jdinius-dev** [comment]
Those do account for the majority of my fails, but not all. 3 Other tests failed : UAE_Dnwind_YRamp_WSt, 5MW_Land_BD_DLL_WTurb, and 5MWX_OCETrpd_DLL_WSt_WavesReg. The  error in the first two seems to be comparatively small (just outside the tolerance range) , but the third is much further off (images below for reference). I will report if changing the gfortran version works once it finishes. 
![Screenshot_2020-07-15 5MW_OC3Trpd_DLL_WSt_WavesReg Summary](https://user-images.githubusercontent.com/66981721/87602683-aa316c00-c6b4-11ea-8b03-43ae85fe6a2b.png)
![Screenshot_2020-07-15 5MW_Land_BD_DLL_WTurb Summary](https://user-images.githubusercontent.com/66981721/87602690-adc4f300-c6b4-11ea-92ef-a2270223e80a.png)
![Screenshot_2020-07-15 UAE_Dnwind_YRamp_WSt Summary](https://user-images.githubusercontent.com/66981721/87602697-b0bfe380-c6b4-11ea-8ff8-3c37fadbffd3.png)

**Comment by jdinius-dev** [comment]
OK I reverted to gfortran-7 and gcc-7 and it resolved many of the tests. Now only four our failing, and none are as far off. A screenshot of the final output is below, along with one of the failed test's plots
![image](https://user-images.githubusercontent.com/66981721/87733879-3fecfa00-c78e-11ea-83a1-69f91128ed10.png)
![image](https://user-images.githubusercontent.com/66981721/87734052-bd186f00-c78e-11ea-91ea-8a98c93ab93a.png)
I noticed that a common through-line of the failed tests was they all have either Wst or Wturb in the title. Screenshots of my ccmake configuration ar also provided in case this is a result of an incorrect configuration or me reverting the gcc version incorrectly. It should be noted that I was unable to revert to gcc 7.4 as it's not in the Ubuntu repositories and I could not find it online. gcc 7.5.0 is being used. 
![image](https://user-images.githubusercontent.com/66981721/87734266-5e9fc080-c78f-11ea-9a89-da1817fd7fe1.png)
